docs/research/prototypes/py2puml-seqdiagram/seqdiagram/kb_entrypoint_fixed.html

```
@@ -0,0 +1,26 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sequence Diagram</title>
+    <script type="module">
+        import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.esm.min.mjs';
+        mermaid.initialize({ startOnLoad: true });
+    </script>
+</head>
+<body>
+    <!-- Your Mermaid diagram -->
+    <div class="mermaid">
+        sequenceDiagram
+        kb_entrypoint_diagram_fixed.py->>bool:module()--build_and_report(True)
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)